include/gpt.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <glib.h>
+
+#include "update_handler.h"
+
+/**
+ * Get the address and size of the inactive boot partition
+ * if a partition exists in the defined region.
+ *
+ * @param device dev path (/dev/mmcblkX)
```

Comment:
In the forthcoming commit 94a310115eea you explicitely used `/dev/sda` in the corresponding section of `system.conf`. So perhaps use `/dev/sdX` instead of `/dev/mmcblkX` here as well? The same applies to line 29 in this file.